### PR TITLE
Update the Cyberith SDK documentation

### PR DESCRIPTION
**Pull request by kouroshD**
With this PR, we add the documentation for adding the user environment variable for the new version of the cyberith sdk of the treadmill.

### CODE REVIEW

File: docs/Dependencies.md [hunk 1]
```diff
@@ -162,9 +162,17 @@
...
      
-   2. `CybSDK`: you can download the application from [this link](https://developer.cyberith.com/downloads). Go to `SDK` tab, download the `C++ SDK for Windows (Developed for: Visual Studio 2015 Community)`. Copy the download application to the workspace. in theory, we should build it in our Windows machine, but time being it is already built (there exists the .dll file) and is working, so you do not need to build it.
+   2. `CybSDK`: you can download the application from [this link](https://developer.cyberith.com/downloads). Go to `SDK` tab, download the `C++ SDK for Windows (Developed for: Visual Studio 2015 Community)`. Copy the download application to the workspace.
    
-      - Append the following variables to the User environmental variable `Path`:
+     - **If you are using version later than [v0.2.0](https://github.com/robotology/walking-teleoperation/tree/v0.2.0)**: After Downloading the SDK, append the following variables to the user environmental variable:
```
Comment (S-Dafarra) [inline docs/Dependencies.md:167]: In principle, you can avoid this kind of comments if you are changing directly the README of the 2.0 version. I guess it is ok also like this at the moment. We just have to remember to remove this comment once ``devel`` gets merged in ``master``.

**Review by S-Dafarra — APPROVED** [review]

File: docs/Dependencies.md (hunk 1 above)
Comment (kouroshD) [reply docs/Dependencies.md:167]: Actually, you are right. Also, I should have opened the PR on top of `devel`, not the `master`. So, I will update the PR.

Comment (kouroshD) [reply docs/Dependencies.md]: @S-Dafarra  updated, please have a look.

File: docs/Dependencies.md [hunk 2]
```diff
@@ -162,9 +162,8 @@
...
      
-   2. `CybSDK`: you can download the application from [this link](https://developer.cyberith.com/downloads). Go to `SDK` tab, download the `C++ SDK for Windows (Developed for: Visual Studio 2015 Community)`. Copy the download application to the workspace. in theory, we should build it in our Windows machine, but time being it is already built (there exists the .dll file) and is working, so you do not need to build it.
-   
-      - Append the following variables to the User environmental variable `Path`:
-        ```     
-        Path: <path to codes workspace>\CybSDK\src\CybSDK_Windows_Cpp_Console\Plugins\x86_64
-        ```
+   2. `CybSDK`: you can download the application from [this link](https://developer.cyberith.com/downloads). Go to `SDK` tab, download the `C++ SDK for Windows (Developed for: Visual Studio 2015 Community)`. Copy the download application to the workspace. After Downloading the SDK, append the following variables to the user environmental variable:
```
Comment (S-Dafarra) [inline docs/Dependencies.md:165]: Is 
```
Copy the download application to the workspace.
```
needed?

**Comment by kouroshD** [comment]
@S-Dafarra  if you agree, I merged it!?

**Comment by kouroshD** [comment]
> I just spotted this, is up to you. For me you can also merge it 😉

I've updated that. I'll megre it.

**Pull request merged**